symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/stream/OboStreamService.java

```diff
@@ -0,0 +1,53 @@
+package com.symphony.bdk.core.service.stream;
+
+import com.symphony.bdk.core.api.invoker.ApiException;
+import com.symphony.bdk.core.api.invoker.ApiRuntimeException;
+import com.symphony.bdk.core.auth.AuthSession;
+import com.symphony.bdk.gen.api.StreamsApi;
+import com.symphony.bdk.gen.api.model.StreamAttributes;
+import com.symphony.bdk.gen.api.model.StreamFilter;
+import com.symphony.bdk.gen.api.model.V2StreamAttributes;
+
+import java.util.List;
+
+class OboStreamService {
+
+  protected final StreamsApi streamsApi;
+
+  protected OboStreamService(StreamsApi streamsApi) {
+    this.streamsApi = streamsApi;
+  }
+
+  /**
+   * {@link StreamService#getStreamInfo(String)}
+   *
+   * @param oboSession  Obo Session
+   * @param streamId    The stream id
+   * @return            The information about the stream with the given id.
+   * @see               <a href="https://developers.symphony.com/restapi/reference#stream-info-v2">Stream Info V2</a>
+   */
+  public V2StreamAttributes getStreamInfo(AuthSession oboSession, String streamId) {
```

Comment:
IMHO this is misleading as `oboSession` can actually be a bot sesssion.

Reply:
Okay I will rename it. 
(But IMHO, the bot session is actually the obo session that the bot do a call on behalf of itself 😄 )